Realistic_Electric_Trains_0.3.0/logic/events/on_tick.lua

```diff
@@ -9,16 +9,30 @@ do
 	function on_tick(event)
 
 		if not loco_fuel_data then
+			local loco_1_prototype = game.entity_prototypes["ret-electric-locomotive"]
+			local loco_1_efficiency_modifier = 1.05
+			if loco_1_prototype.burner_prototype.effectivity < 1 then
+				loco_1_efficiency_modifier = loco_1_efficiency_modifier * (2 - loco_1_prototype.burner_prototype.effectivity)
```

Comment:
One question: Why do you calculate it like this? I'd have expected something along these lines:
```suggestion
				loco_1_efficiency_modifier = loco_1_efficiency_modifier / loco_1_prototype.burner_prototype.effectivity
```

Reply:
The two formulas have different results. Mine adds just the extra energy required to compensate for efficiency. Yours raises a bit faster, offering more transfer capacity.
I see no problem using a formula resulting in a larger transfer buffer.